### ISSUE TITLE
Unsupported keras/tensorflow version

### ISSUE THREAD

**Issue by wedoso** [deeplearning4j/deeplearning4j#9693]
#### Issue Description
There is an issue that dl4j won’t properly load keras model if the version of keras or tensorflow is not supported. My current investigation is beta7 can only support the version up to keras=2.4.3 and tensorflow=2.2.0. Is it the true statement?

#### Version Information

Please indicate relevant versions, including, if relevant:

* Deeplearning4j version: beta7
* Platform information (OS, etc): macOS 12.3.1
* CUDA version, if used: NA
* NVIDIA driver version, if in use: NA

#### Additional Information
I provide the simple functional model for the reference but it is not necessary to be the model like below 

    import keras
    inputs = keras.layers.Input(shape=(237,))
    x = inputs
    for i in range(4):
        x = keras.layers.Dropout(rate=0.5)(x)
        x = keras.layers.Dense(units=int(237)/(2**(i+1)), activation='relu')(x)
    outputs = keras.layers.Dense(200, activation='sigmoid')(x)
    model = keras.Model(inputs=inputs, outputs=outputs, name="test_model")
    model.compile(optimizer='adam',
                  loss='categorical_crossentropy',
                  metrics=['categorical_crossentropy', 'mse'])
    print(model.summary())

You may try above script with tensorflow>2.2.0 and keras=2.4.3 and save the model. This saved h5 file can’t be properly loaded via dl4j.

#### Contributing

If you'd like to help us fix the issue by contributing some code, but would
like guidance or help in doing so, please mention it!

If the statement I mentioned above is true, please add the supported keras and tensorflow version to the documentation to avoid inconvenience for the users.

**Comment by agibsonccc** [comment]
Closing this. There's been multiple new versions since beta7 at this point. We've had keras support for a long time now. The main missing feature are the new normalization layers. We also support TF import. If you want, please consider opening an issue for something more specific.
